frontend/src/pages/admin/Template/index.vue

```diff
@@ -125,18 +125,17 @@ export default {
                             changed = changed || pmChanges.changed
                             errors = errors || pmChanges.errors
                         } else if (field === 'palette_catalogue') {
-                            this.editable.changed.settings.palette_catalogue = false
```

Comment:
Previously this was set to false, then toggled to true if the palette_catelogue has changed, this meant that after the first watch handler had fired and was watching `this.editable.changed` too, the object would _always_ mutate.

Setting it only once means it does not mutate.